doofinder-for-woocommerce/includes/settings/class-register-settings.php

```diff
@@ -131,7 +131,24 @@ function () use ($api_host_option_name) {
             ]
         );
 
-        register_setting(self::$top_level_menu, $api_host_option_name, array($this, 'validate_api_host'));
+        register_setting(self::$top_level_menu, $api_host_option_name, array($this, 'validate_hosts'));
```

Comment:
I think we should have only one server setting and choose between "us" and "eu". Then we could use that for building the endpoints.

Reply:
I understand, and I think I agree.
I will write it down to study this refactor and implement it in the future!